source/fem-dem/vans_assemblers.cc

```diff
@@ -1093,21 +1093,21 @@ GLSVansAssemblerKochHill<dim>::calculate_particle_fluid_interactions(
                 16.14 * (1 - cell_void_fraction)) /
                (1 + 0.681 * (1 - cell_void_fraction) -
                 8.48 * pow(1 - cell_void_fraction, 2) +
-                8.14 * pow(1 - cell_void_fraction, 3));
+                8.16 * pow(1 - cell_void_fraction, 3));
         }
       else if ((1 - cell_void_fraction) >= 0.4)
         {
           f0 = 10 * (1 - cell_void_fraction) / pow(cell_void_fraction, 3);
         }
 
       double f3 = 0.0673 + 0.212 * (1 - cell_void_fraction) +
-                  0.0232 / pow(cell_void_fraction, 5);
+                  0.0232 * pow(cell_void_fraction, -5);
```

Comment:
why this change?

Reply:
I think I would keep the division here, just because it is easier to compare with the article.

Reply:
Done